### DISCUSSION TITLE
[Help]add select tag

### DISCUSSION THREAD

**Discussion by xiu-xiao** [windingwind/zotero-actions-tags#188] [Q&A] [answered]
### Description

求助一下可不可以自己选择添加那些标签然后弄个选项
![WechatIMG274](https://github.com/windingwind/zotero-actions-tags/assets/150535352/b22a2ff3-2c87-41f7-bfaa-9bc13eed1aca)

### Event

None

### Operation

None

### Data

```javascript
/**
 * A description of this script.
 * @author Zotero Community
 * @usage 
 * @link https://github.com/windingwind/zotero-actions-tags/discussions/
 * @see https://github.com/windingwind/zotero-actions-tags/discussions/
 */
```

**Comment by windingwind** [comment, marked as answer]
抱歉我不理解你的问题

**Reply by xiu-xiao** [reply]
<img width="207" alt="截屏2023-11-13 16 48 50" src="https://github.com/windingwind/zotero-actions-tags/assets/150535352/5a2972fa-b065-4f8c-b819-71103f7932c7">
就是我想弄一个，可以每次我想添加标签的时候就给我提供一些选项（例如我图片中的这几个图标），可以让我选择，类似于做选择题

**Reply by windingwind** [reply]
zo的标签输入框有自动补全。你可以给所有标签前面加一个同样的字符（例如`#`, `@`），每次输入对应的首字符，后面的就会候选出来的了

**Reply by xiu-xiao** [reply]
可以可以感谢感谢
